core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
         update:
           apiServer:
             certSANs: >-
-              {{ groups['kubernetes_master'] | map('extract', hostvars, ['ansible_default_ipv4', 'address']) | list
+              {{ (groups['kubernetes_master'] | map('extract', hostvars, ['ansible_default_ipv4', 'address']) | list ) + (groups['kubernetes_master'] | map('extract', hostvars, ['ansible_host']) | list ) | unique
```

Comment:
certSANs are also in core\src\epicli\data\common\ansible\playbooks\roles\kubernetes_master\templates\kubeadm-config.yml.j2

Reply:
I'm not sure if in core\src\epicli\data\common\ansible\playbooks\roles\kubernetes_master\templates\kubeadm-config.yml.j2
we should also change it. @

Reply:
I think for consistency reasons we should manage both places the same exact way, so I would update kubeadm config as well. The one example of what happened when we were updating running config and not kubeadm one was with etcd encryption, it has caused so much drama during upgrades that I would not want this to be repeated ever.

Reply:
I made the fix also for kubeadm-config.yaml.j2 :) done.